azure-iot-device/azure/iot/device/common/transport_exceptions.py

```diff
@@ -38,3 +38,11 @@ class ProtocolClientError(ChainableException):
     """
 
     pass
+
+
+class PipelineTimeoutError(ChainableException):
```

Comment:
Seems that this should be in `transport_exceptions.py` rather than `pipeline_exceptions.py`?